coursedata/pages/for-teachers/en.yaml

```diff
@@ -0,0 +1,91 @@
+title: "Hedy documentation"
+sections:
+    introduction:
+        title: "Introduction"
+        text:
+            whats_hedy:
```

Comment:
Do we need these names (like `whats_hedy`) hard coded into the yaml? We can also simply make a list of key, text pairs? That gives us a bit more flexibility and less duplication. I see in the HTML template we do not really use these names anyway.

Otherwise, really good idea this way!!

Reply:
I changed the keys to numeric values, simply counting up with each (sub)sections!